src/main/java/org/jabref/logic/formatter/bibtexfields/RemoveEnclosingBracesFormatter.java

```diff
@@ -15,7 +15,7 @@ public String getName() {
 
     @Override
     public String getKey() {
-        return "remove_braces";
```

Comment:
I think, this is the "old" formatter?

Since it is used in `.layout` files, please don't change the name. Otherwise, we need to add support for alias names....

Reply:
Oh, I could not find references to this key anywhere in the code so I made this more descriptive. Reverting this